src/Converter/UnitMeasurementValueConverter.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+
+namespace Webgriffe\SyliusAkeneoPlugin\Converter;
+
+use Webgriffe\SyliusAkeneoPlugin\MeasurementFamiliesApiClientInterface;
+use Webmozart\Assert\Assert;
+
+final class UnitMeasurementValueConverter implements UnitMeasurementValueConverterInterface
+{
+    public const RECOGNIZED_OPERATORS = ['add', 'sub', 'mul', 'div'];
```

Comment:
```suggestion
    private const RECOGNIZED_OPERATORS = ['add', 'sub', 'mul', 'div'];
```

Reply:
This constant is public because it is also used outside this class.

Reply:
It's used in the spec only. If the only reason to have it public is to use it in a spec it sounds a code smell...